### DISCUSSION TITLE
Util.telescope is not undefined

### DISCUSSION THREAD

**Discussion by danielo515** [LazyVim/LazyVim#3910] [Q&A] [answered]
I have been looking for information about the removal of this, but I can't find any, and the worse, I can't find the alternative to use.
Can someone please point me in the right direction?

**Comment by dpetka2001** [comment, marked as answer]
Try `LazyVim.pick` instead.

**Reply by danielo515** [reply]
Ok, how do I bring LazyVim into scope? I mean, what do I require?

**Reply by folke** [reply]
LazyVim is always defined

**Reply by danielo515** [reply]
Ok, yes, seems to be magically in scope. Thanks
